docs/user/openstack/customization.md

```diff
@@ -137,6 +138,10 @@ platform:
       clusterOSImage: my-rhcos
 ```
 
+## Custom Subnets
+
+If you want to install your cluster on a custom network and subnet, you can pass the ID of the subnet to the install config with the `machinesSubnet` option. This subnet will be tagged as the primary subnet for the cluster. The nodes, and VIP port will be created on this subnet. This does add some complexity to the install process that needs to be accounted for. First of all, the subnet must have DHCP enabled. Also, the CIDR of the custom subnet must match the cidr of `networks.machineNetwork`. Be aware that the API and Ingress VIPs by default will try to take the .5 and .7 of your network CIDR. If they are not available, then you should set them in the install config by setting the `apiVIP` and `ingressVIP` options to the IP addresses you want to reserve. Lastly, note that setting `externalDNS` while setting `machinesSubnet` is invalid usage, since the installer will not modify your subnet. If you want to add a DNS to your cluster while using a custom subnet, add it to the subnet in openstack [like this](https://docs.openstack.org/neutron/rocky/admin/config-dns-res.html).
```

Comment:
I'd like a comment somewhere saying that `apiVIP` and `ingressVIP` should be picked outside of the DHCP allocation range in order to avoid any IP address conflicts.

Reply:
hmm yeah that is wise, I will add that